app/views/courses/_contact_details.html.erb

```diff
@@ -2,30 +2,30 @@
   <h2 class="govuk-heading-l" id="section-contact">Contact details</h2>
 
   <div class="course-basicinfo">
-    <dl class="govuk-list--description">
+    <dl class="app-list--description">
```

Comment:
Is this a app-specific component, or a modification of a `govuk-frontend` one? I don't think `govuk-list` is meant to be used with definition lists?

Reply:
Good point. Updated this from `govuk-list--description` to `app-description-list`